### ISSUE TITLE
APICheck fails with ReflectionTypeLoadException on .NET Core 2.1

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/BuildTools#492]
This failure was introduced with https://github.com/aspnet/KestrelHttpServer/commit/ce684270500a12ea25e8f86164fbe082e4eeda13, however, it doesn't reproduce on all machines.
```
[10:02:39]  KestrelHttpServer     |   System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
[10:02:39]  KestrelHttpServer     |     System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
[10:02:39]  KestrelHttpServer     |     System.IO.FileNotFoundException: Could not load the specified file.
[10:02:39]  KestrelHttpServer     |     System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
[10:02:39]  KestrelHttpServer     |     System.IO.FileNotFoundException: Could not load the specified file.
[10:02:39]  KestrelHttpServer     |     --- End of inner exceptions ---
[10:02:39]  KestrelHttpServer     |     --- End of loader exceptions ---
[10:02:39]  KestrelHttpServer     |      at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
[10:02:39]  KestrelHttpServer     |      at System.Reflection.RuntimeAssembly.get_DefinedTypes()
[10:02:39]  KestrelHttpServer     |      at ApiCheck.ApiListingGenerator.GenerateApiListing() in C:\b\w\9571b3e5f03d051f\src\ApiCheck.Console\ApiListingGenerator.cs:line 41
[10:02:39]  KestrelHttpServer     |      at ApiCheck.Program.OnCompare(CommandLineApplication command, CommandOption apiListingPathOption, CommandOption breakingChangesPathOption, CommandOption assemblyPath, CommandOption assetsJson, CommandOption framework, CommandOption excludeInternalNamespace) in C:\b\w\9571b3e5f03d051f\src\ApiCheck.Console\Program.cs:line 212
[10:02:39]  KestrelHttpServer     |      at ApiCheck.Program.<>c__DisplayClass2_1.<Main>b__3() in C:\b\w\9571b3e5f03d051f\src\ApiCheck.Console\Program.cs:line 69
[10:02:39]  KestrelHttpServer     |      at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args) in C:\b\w\9571b3e5f03d051f\shared\Microsoft.Extensions.CommandLineUtils.Sources\CommandLine\CommandLineApplication.cs:line 293
[10:02:39]  KestrelHttpServer     |      at ApiCheck.Program.Main(String[] args) in C:\b\w\9571b3e5f03d051f\src\ApiCheck.Console\Program.cs:line 81
[10:02:39]  KestrelHttpServer     | C:\b\w\c6765912498f135c\.packages\internal.aspnetcore.sdk\2.1.0-preview1-15551\build\ApiCheck.targets(39,5): error MSB6006: "dotnet.exe" exited with code 1.
```
http://aspnetci/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=359729&_focus=6414

**Comment by natemcmaster** [comment]
Assigning @ryanbrandenburg since he is still looking into this

**Comment by natemcmaster** [comment]
When fixed, we should revert https://github.com/aspnet/KestrelHttpServer/pull/2185

**Comment by ryanbrandenburg** [comment]
Fixed by aspnet/Universe#677. Will revert aspnet/KestrelHttpServer#2185.
